uaclient/status.py

```diff
@@ -146,8 +152,8 @@ class UserFacingStatus(enum.Enum):
 
 STATUS_UNATTACHED_TMPL = "{name: <14}{available: <11}{description}"
 
-STATUS_SERVICE_HEADER = "\nSERVICE"
-STATUS_TMPL = "{name: <14}{entitled: <26}{status}"
+STATUS_HEADER = "SERVICE       ENTITLED  STATUS    DESCRIPTION"
```

Comment:
Can we do the same trick as before, and define the header in terms of the template?

Reply:
We can't do the same trick here because of the unprintable characters we emit due to our colorize calls, it adds about 5 unprintable characters that mess with the .format alignment if we stuff the header strings directly into the formatted template

Reply:
Ah, bummer. In that case, could we get a comment on the template explaining why the numbers contained within it aren't actually the widths we'll see in the output? That seems like something that would be very easy to trip over in future.